cli_tools/common/utils/logging/service/log_entry.go

```diff
@@ -254,6 +254,21 @@ type OutputInfo struct {
 	IsUEFICompatibleImage bool `json:"is_uefi_compatible_image,omitempty"`
 	// Indicates whether the image is auto-detected to be UEFI compatible
 	IsUEFIDetected bool `json:"is_uefi_detected,omitempty"`
+	// InspectionResults contains metadata determined using automated inspection
+	InspectionResults InspectionResults `json:"inspection_results,omitempty"`
+}
+
+// InspectionResults contains metadata determined using automated inspection
+type InspectionResults struct {
+	// UEFIBootable indicates whether the disk has a UEFI boot loader.
```

Comment:
This field should indicate whether the identified OS is bootable with UEFI. The next field is similar, but should indicate whether it's bootable with BIOS